solutions/tests/test_mean.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import unittest
+from solutions.mean import mean
+
+
+class TestMean(unittest.TestCase):
+    """test the mean function"""
+
+    def test_1(self):
```

Comment:
Using descriptive test names, such as test_mean_3 instead of generic names like test_1, can make the purpose of the test clearer and more helpful for understanding.

Reply:
okay i will get it done